### ISSUE TITLE
Thursday

### ISSUE THREAD

**Issue by Codingmaestre2015** [WestMEC-Coding-Session2/Week-28-Bell-Work#3]

**Comment by shumway-kyle-wm** [comment]
yes you can. an example being;
``` html
<p> how has your day been <php? echo $name ?>? </p>
```

**Comment by murillo-daisy-wm** [comment]
PHP will work as long as its in it's own tags ( <?php ... ?> ), so I'm guessing it can be embedded with an HTML line.

**Comment by casselman-bailey-wm** [comment]
PHP tags can be anywhere in the html file as long as it is defined as such.

**Comment by warren-evan-westmec** [comment]
It has to be its own line
 <?php

?>

**Comment by chavez-juan-wm** [comment]
You can embed PHP in an HTML line, you just have to add "<?php ?>"  within the HTML where you want to add the PHP.

**Comment by cantrell-tyler-wm** [comment]
You can embed it inside html you just put a <?php....?>

**Comment by spiller-taylor-wm** [comment]
PHP just has to be enclosed by <?php ?>. It can be nested inside of any tag.

**Comment by ErickCiudad** [comment]
It can work, for example: <p <?php ?>> You could run a php script inside a paragraph</p>

**Comment by yousif-zues-wm** [comment]
PHP is in the position of an HTML attribute as an attribute is in the position after the declaration of the tag EG: ```<div```  <b> id=" headOne"</b>>
PHP EG: <?php '''  <b>  $user_name = $_POST ["username"] </b> ?>

**Comment by do-vuong-wm** [comment]
It can be apart of the HTML tag attribute because you can add <?php ?>.

**Comment by chopra-austin-west-mec** [comment]
php can be within html line but should be on its own separate line in order to read it better.

**Comment by chu-christopher-wm** [comment]
It can be both, using <?php...?> and methods and actions throughout the PHP

**Comment by kurtz-zachary-wm** [comment]
PHP can only work in HTML if it is part of its own tag.

**Comment by adenola-ahmed-wm** [comment]
Yes. PHP can work inside the HTML you just have to put the correct tag for PHP

**Comment by lopez-efrain-wm** [comment]
It can be anywhere as long as it is inside ``` <? php ?> ```

**Comment by turner-dalton-wm** [comment]
Yes, you can embed php code to html attributes:
`
<input value="<?php echo phpversion();?>">
`

**Comment by iniguez-angel-wm** [comment]
PHP has it's own line. Much like how javascript and CSS has its own line. 
``<?php   ?>``

**Comment by navarro-aiden-wm** [comment]
You can embed it in an HTML line  as part of an HTML tag attribute. 
``<?php ?>``

**Comment by moncada-carlos-WM** [comment]
A PHP code can be embedded in an HTML because the code goes inside the body of an HTML page.

**Comment by Tucker-Cole-wm** [comment]
PHP can be in a line containing HTML as long as it has its own tags.
